examples/billboard/billboard.py

```diff
@@ -0,0 +1,187 @@
+    """.format(
+        view_id,source_id
+    )
+
+    bq_view_client = bigquery.Client(project=args.PROJECT_ID) #not sure why this need project_id
+
+    job = bq_view_client.query(sql)  # API request.
+    job.result()  # Waits for the query to finish.
+
+    if isStandard == True:
+        output_url=report_base_url+standard_view_url.format(args.PROJECT_ID, args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_standard)
+    else:
+        output_url=output_url+detailed_view_url.format(args.PROJECT_ID, args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_detailed)
+ 
+    print(
+        'Created view {}{}.{}.{}'.format(Back.GREEN,
+            job.destination.project,
+            job.destination.dataset_id,
+            job.destination.table_id
+        )
+    )
+    print(Style.RESET_ALL)
+
+def generate_datastudio_url(args):
+    print("To view dataset, please click "+Back.GREEN +"https://console.cloud.google.com/bigquery","\n")
+
+    print(Style.RESET_ALL)
+
+    print("To launch datastudio report, please click " +Back.GREEN + output_url +"\n")
+    print(Style.RESET_ALL)
+
+def remove_billboard_dataset(args):
+    standard_view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_standard)
+    bq_client.delete_table(standard_view_id, not_found_ok=True)
+    print("Billboard view {} deleted.".format(standard_view_id))
+    detailed_view_id="{}.{}.{}".format(args.PROJECT_ID,args.BILLBOARD_DATASET_NAME_TO_BE_CREATED,args.bb_detailed)
+    bq_client.delete_table(detailed_view_id,not_found_ok=True)
+    print("Billboard view {} deleted.".format(detailed_view_id))
+    return True
+
+
+def main(argv):
+
+   parser = argparse.ArgumentParser(description='Billing Export information')
+   parser.add_argument('-pr', dest='PROJECT_ID', type=str,help='Project Id', required=True)
+   parser.add_argument('-se', dest='STANDARD_BILLING_EXPORT_DATASET_NAME', type=str,required=True)
+   parser.add_argument('-de', dest='DETAILED_BILLING_EXPORT_DATASET_NAME', type=str)
+   
+   parser.add_argument('-bb', dest='BILLBOARD_DATASET_NAME_TO_BE_CREATED', type=str,required=True)
+   
+   parser.add_argument('-clean', dest='clean', type=str, help='Only when you need cleanup, provide "yes"')
+   
+
+   args = parser.parse_args()
+   if args.DETAILED_BILLING_EXPORT_DATASET_NAME == None:
+       args.DETAILED_BILLING_EXPORT_DATASET_NAME = args.STANDARD_BILLING_EXPORT_DATASET_NAME
+
+   project_id_temp="projects/{}".format(args.PROJECT_ID)
+   try:
+        project_billing_info =billing.CloudBillingClient().get_project_billing_info(name=project_id_temp) 
+   except PermissionDenied:
+        print("Permission Denied so you do not have project level permission or provided wrong project id, please check.")
+        return False
```

Comment:
To indicate that the program has finished unsuccessfully, you can use `return sys.exit(1)`

Reply:
Modified as per standard